### PR TITLE
Heat Orchestration generating script for openstack

### PR DESCRIPTION
**Pull request by StLuke**
Adding also flake8 support for python-scripts to tox

### CODE REVIEW

File: python-scripts/generate_hot_tempalte.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    parser.add_argument(
+        '--remote-db', action='store_true',
+        default=False, help='New instance for remote db'
+    )
+    parser.add_argument(
+        '--remote-dwh-db', action='store_true',
+        default=False, help='New instance for remote dwh db'
+    )
+    parser.add_argument(
+        '--file', '-f', default='/tmp/hot.yaml',
```
Comment (grafuls) [inline python-scripts/generate_hot_tempalte.py]: I would get rid of the default value here if we can't be sure the file will be there and make it mandatory

File: python-scripts/generate_hot_tempalte.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        args.prefix + "_remote_dwh_db" + suffix
+    ]:
+        if i in hot_template['resources']:
+            hot_template['resources'].pop(i)
+            hot_template['outputs'].pop(i+'_public_ip')
+    if args.delete:
+        with open(args.file, 'w') as f:
+            yaml.dump(hot_template, f, default_flow_style=False)
+        exit(1)
+    for i, s in enumerate(instances):
```
Comment (grafuls) [inline python-scripts/generate_hot_tempalte.py]: I would personally use a list comprehension here:
instances = [s + suffix for s in instances]

**Comment by grafuls** [comment]
Looks good to me +1

File: python-scripts/generate_hot_tempalte.py (hunk 1 above)
Comment (StLuke) [inline python-scripts/generate_hot_tempalte.py]: done

File: python-scripts/generate_hot_tempalte.py (hunk 2 above)
Comment (StLuke) [inline python-scripts/generate_hot_tempalte.py]: done

**Pull request merged**